docs/src/starknet/script.md

```diff
@@ -0,0 +1,176 @@
+# Cairo Deployment Scripts
```

Comment:
Let's add something about using `path_to_scarb_toml` when running `sncast script` from arbitrary directory

Reply:
We mention this in several places in the docs, but never in sections dedicated to specific subcommands. Do you think we should change that? I guess this is rather top-level stuff

Reply:
okay, it can stay as is then